lib/models/application.ts

```diff
@@ -940,6 +940,7 @@ const getApplicationModel = function (
 		 * @memberof balena.models.application
 		 *
 		 * @param {String|Number} nameOrSlugOrId - application name (string) (deprecated), slug (string) or id (number)
+		 * @param {String} keyName - Provisioning key name (string) - [OPTIONAL]
```

Comment:
```suggestion
		 * @param {String} [keyName] - Provisioning key name
```
The docs generator should do the rest :)